### ISSUE TITLE
Welcome Page for the Site

### ISSUE THREAD

**Issue by vidit624** [rsrkpatwari1234/EchoForMe#1]
Create a welcome page for the site mentioning the purpose of this project

**Comment by Amit366** [comment]
Is it something like splash screen?

**Comment by barnik109** [comment]
I want to take this issue.

**Comment by rsrkpatwari1234** [comment]
It is just like creating a good opening page displaying the purpose of the site.Since when we will deploy it,the viewer shall have no idea of the usage. You can find the purpose in the readme.

**Comment by rsrkpatwari1234** [comment]
@barnik109 please let me know if you want this issue under jwoc or dwoc?

**Comment by barnik109** [comment]
Under jwoc.

**Comment by rsrkpatwari1234** [comment]
@barnik109 I will assign you this issue,once the coding phase starts

**Comment by barnik109** [comment]
This can be created by html and css only??

**Comment by barnik109** [comment]
Pleass don't assign me until i set up the project as i am having some problems.I will inform when the project is set up and them you can assign me.

**Comment by barnik109** [comment]
*then

**Comment by Swastik1710** [comment]
Hey @vidit624 , can I work on this issue ?

**Comment by vidit624** [comment]
@Swastik1710 Sure go ahead !!

**Comment by Swastik1710** [comment]
Do you have any awesome background image for this.
Btw I'm thinking of using React for this .

**Comment by rsrkpatwari1234** [comment]
@Swastik1710 I do not think that for creating a simple page, we should include a new language. However you are free to use html,css,php and js for adding animations.

As far as background is concerned, you can see the demo video and the current background image. We do not have any choices as of  now, but we want the welcome page to hold similar textures and color composition.

**Comment by Swastik1710** [comment]
Actually my idea was to create a website with the help of which you can showcase your News reader.
The website will have a tab named Listen News and the entire app backend will work under this. 

Anyways, please have a look at the screenshots of the website shared in the discord channel. 
 
Also, React makes the website pretty smooth with the use of React Router as the pages are preloaded and hence on clicking any of the pages, the browser or the page doesn't reload and it goes directly to the desired page.

**Comment by Swastik1710** [comment]
Yeah okay @rsrkpatwari1234 
Got your point.
I can do it with html , css and javascript .

**Comment by jatinjain001** [comment]
I would like to work on it

**Comment by jatinjain001** [comment]
I am working on it. but I am getting error related to database connections 
![Screenshot (210)](https://user-images.githubusercontent.com/46483236/106637311-867a3880-65a8-11eb-8073-1152c96c6b68.png)

**Comment by vidit624** [comment]
@jatinjain001 kindly post your doubts on the discord channel. We'll try to help you there

**Comment by kanishkaa24** [comment]
Hello! Can I work on this?
